src/main/java/frc/trigon/robot/subsystems/arm/ArmConstants.java

```diff
@@ -0,0 +1,114 @@
+package frc.trigon.robot.subsystems.arm;
+
+import com.ctre.phoenix6.hardware.CANcoder;
+import com.ctre.phoenix6.hardware.TalonFX;
+import com.revrobotics.CANSparkMax;
+import com.revrobotics.CANSparkMaxLowLevel;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Rotation2d;
+
+public class ArmConstants {
```

Comment:
In this class, you named some of the motors neo and some spark max. A spark max is a **motor controller** for a neo (and other motors). Hence, there is no difference. You should name one a `MASTER` and the other one `FOLLOWER` for every pair of motors.